README.rst

```diff
@@ -67,15 +67,8 @@ Some of our dependencies are hard to install, so it's best to use ``conda``::
     conda create -n nwb2 python=3.6 pip numpy pandas hdf5 h5py
     conda install -n nwb2 av tifffile -c conda-forge
     conda activate nwb2
-    pip install .[video]
-
-Eventually the last line will be replaced with::
-
     pip install silverlabnwb[video]
 
-But we haven't uploaded to PyPI yet.
-
-
```

Comment:
I would leave in one of these, so that there are two blank lines (as with the other segments). I don't know if it makes a difference in how it's displayed, but it helps when viewing the raw text, at least.